### PR TITLE
Created anaconda package

### PR DESCRIPTION
**Pull request by Elizabeth2507**

### CODE REVIEW

**Comment by thomwolf** [comment]
I've just released a new version 4.0 based on SpaCy 2.1+ with a simpler dependency structure. It's now also on PyPI.
I'm closing this for now but maybe we can investigate a conda package again in the future.

**Comment by huanyu-li** [comment]
Hi, Is this command "conda install -c laed neuralcoref" working for Mac OS? Thanks!

**Comment by Elizabeth2507** [comment]
Hello

No, it's a version for Linux linux-64/neuralcoref-0.1-py36ha79b42c_0.tar.bz2<https://anaconda.org/laed/neuralcoref/0.1/download/linux-64/neuralcoref-0.1-py36ha79b42c_0.tar.bz2>
________________________________

File: readme.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ This coreference resolution module is based on the super fast [spaCy](https://sp
 Be sure to check out [our medium post](https://medium.com/huggingface/state-of-the-art-neural-coreference-resolution-for-chatbots-3302365dcf30) in which we talk more about neuralcoref and coreference resolution.
 
 ## Installation
+Method #1
+It's available to install neuralcoref package from Anaconda.org:
+In Terminal window or Anaconda Prompt, run:
+
+```
+conda install -c laed neuralcoref
```
Comment (FHChohan) [inline readme.md]: Certain packages are not available on the installation

**Pull request closed without merging**